src/components/Common/ButtonText.tsx

```
@@ -0,0 +1,130 @@
+  const getSizeStyle = () => {
+    switch (size) {
+      case ButtonTextSize.SM:
+        return 'body-14-medium py-0.5';
+      case ButtonTextSize.MD:
+      default:
+        return 'button-16-semibold py-1';
+    }
+  };
+
+  // 버튼 타입에 따라 다른 색상을 적용
+  const getColorStyle = () => {
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)